analyzer/tools/compilation-database-transformer/setup.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import setuptools
+
+with open("README.md", "r", encoding="utf-8", errors="ignore") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="compilation database transformer",
+    version="0.1.0",
+    author='CodeChecker Team (Ericsson)',
+    description="Manipulate, preprocess and transform compilation database "
+                "JSON files. Provides pretty printing, compilation result "
+                "checking and adjusting compilation parameters features.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/Ericsson/CompilationDatabaseTransformer",
+    keywords=['compilation_database_transformer', 'codechecker', 'compilation database', 'compile_commands'],
+    license='LICENSE.txt',
+    packages=setuptools.find_packages(),
+    include_package_data=True,
+    classifiers=[
+        "Environment :: Console",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: University of Illinois/NCSA Open Source License",
```

Comment:
Please remove this line because we changed our license to `Apache 2.0 with LLVM Exception` and this is not supported  by this file.